README.md

```diff
@@ -1,13 +1,13 @@
 JMX Exporter
 =====
 
-JMX to Prometheus exporter.
+JMX to Prometheus exporter: a collector that can configurably scrape and
+expose mBeans of a JMX target.
 
-A Collector that can configurably scrape and expose mBeans of a JMX target. It
-meant to be run as a Java Agent, exposing an HTTP server and scraping the local
-JVM.
-
-This can be also run as an independent HTTP server and scrape remote JMX targets.
+This exporter is intended to be run as a Java Agent, exposing a HTTP server
+and serving metrics of the local JVM. It can be also run as an independent
+HTTP server and scrape remote JMX targets, but this has the downside of not
```

Comment:
It has more downsides than that, you also lose process metrics and configuration&management is much more challenging. Most of our support requests are from users needlessly using the http server, so it should be more strongly discouraged.

Reply:
If we're discouraging it so strongly, wouldn't it be smarter to either deprecate this functionality or at least document that it's unsupported?

Reply:
There are valid use cases for it, as some users (largely for political rather than technical reasons) can't use the agent. Thus it remains a supported feature.

Reply:
Got it. What do you think of this instead?